runtime/src/bank.rs

```diff
@@ -1149,6 +1149,11 @@ impl Bank {
         bank.transaction_processor =
             TransactionBatchProcessor::new_uninitialized(bank.slot, bank.epoch);
 
+        if let Some(compute_budget) = bank.compute_budget {
```

Comment:
it seems like basically everywhere we are `new_uninitialized` then this if block.
can we just add a new `new_uninitialized_with_execution_budget`?

Reply:
Except for `_new_from_parent()`, where it's calling `new_from()`.
The call to `set_execution_budget()` will be removed altogether, once we remove `compute_budget` instance from `Bank`. That would be a separate PR though.

Reply:
Moved it to `finish_init()`. So the code is not replicated anymore.